docs/experiments-clueweb12.md

```diff
@@ -1,54 +1,92 @@
-# Anserini Experiments on ClueWeb12
+# Anserini: Experiments on ClueWeb12
 
-Anserini is able to build a single (monolithic) index on all of ClueWeb12:
+## Indexing
+
+Typical indexing command:
 
 ```
 nohup sh target/appassembler/bin/IndexCollection -collection CW12Collection \
- -input /path/to/cw12/ -generator JsoupGenerator \
- -index lucene-index.cw12.cnt -threads 48 -optimize > log.cw12.cnt &
-
+ -input /tuna1/collections/web/ClueWeb12/ -generator JsoupGenerator \
+ -index lucene-index.cw12.pos+docvectors -threads 88 \
+ -storePositions -storeDocvectors \
+ >& log.cw12.pos+docvectors &
```

Comment:
The first `&` should be removed?

Reply:
I actually do mean to use `>&`, see https://www.gnu.org/software/bash/manual/html_node/Redirections.html